common/lib/xmodule/xmodule/modulestore/split_mongo/split_draft.py

```diff
@@ -212,10 +212,13 @@ def delete_item(self, location, user_id, revision=None, **kwargs):
                 branched_location = location.for_branch(branch)
                 parent_loc = self.get_parent_location(branched_location)
                 SplitMongoModuleStore.delete_item(self, branched_location, user_id)
-                # publish parent w/o child if deleted element is direct only (not based on type of parent)
+                # publish parent w/o child if deleted element is direct only (not based on type of parent) or vertical
```

Comment:
Make it clear that you're including verticals to make it more like the old mongo/draft modulestore